src/pages/Question.js

```diff
@@ -0,0 +1,75 @@
+import React, {useState} from 'react'
+import Logo from '../components/Logo.js'
+import Header from '../components/Header.js'
+import  {questions} from '../config';
+import Answer from '../components/Answer.js'
+import '../styles/questions.scss';
+import PreviousButton from '../components/PreviousButton.js';
+
+const Question = () => {
+    const categories = ['food', 'sleep', 'sport', 'misc'];
+    const [currentCategory, setCurrentCategory] = useState(0)
+    const [currentQuestion, setCurrentQuestion] = useState(1)
+
+    let categoryObject = questions[categories[currentCategory]]
+    let questionObject = categoryObject.questions[currentQuestion]
+
+    const nextQuestion = () => {
+
+        setCurrentQuestion(currentQuestion + 1)
+        if (!((currentQuestion+1) in categoryObject.questions)) {
+            if (currentQuestion === 10) {
+                window.location.href = "/results";
+            } else {
+                setCurrentCategory(currentCategory + 1)
+            }
+        }
+    }
+    const previousQuestion = () => {
+        setCurrentQuestion(currentQuestion - 1)
+        if (currentQuestion === 1) {
+            window.location.href = "/";
+        }
+        else if (!((currentQuestion-1) in categoryObject.questions)) {
+            setCurrentCategory(currentCategory - 1)
+        }
+    }
```

Comment:
I think this logic can better be implemented inside the Header component or maybe even App. That way the logic can be written more clean and the results and webcam page can also be controlled via there. Routing will be almost unnecessary.

Reply:
Let's fix this in a seperate PR/issue